MANIFEST.in

```diff
@@ -1,4 +1,6 @@
-recursive-include django_downloadview *
+recursive-include demo *.py *.rst *.html *.json *.txt
```

Comment:
Disclaimer: I'm a newbie in Debian packaging.

First and foremost, I'd appreciate this project being available as a debian package. In order to review the changes, I first need to understand them...

If you need almost all the content of the repository, wouldn't it be simpler to clone the repository? How is the debian package generated?

> I would use it for the tests

At the moment, I don't understand the purpose of such tests. Can you explain it? Or perhaps provide an URL where I could learn about it?

Demo and tests contain stuff that is about the project's development. This stuff can be quite big (data inside). And I'd like users to download only what they actually need... At the moment, demo and tests are excluded from the Python distribution. Would users need demo and tests when they install the debian package?

Note: there is a [django_downloadview.test] (https://github.com/benoitbryon/django-downloadview/blob/master/django_downloadview/test.py) included in the package, as a library to write tests around user-custom views created using django-downloadview.